index.html

```diff
@@ -37,16 +37,166 @@ <h2 class="title">I'm a Software Developer</h2>
             <li><a href="#"><i class="fa-brands fa-facebook fas"></i></a></li>
         </ul>
         <div class="bg-design">
-            <img class="imga" src="img/v7.png">
-            <img class="imgb" src="img/v8.png">
-            <img class="imgc" src="img/v9.png">
-            <img class="imge" src="img/v11.png">
-            <img class="imgf" src="img/V12 (2).png">
-            <img class="imgf1" src="img/V12 (2).png">
-            <img class="imgf2" src="img/V12 (2).png">
-            <img class="imgf3" src="img/V12 (2).png">
-            <img class="imgg" src="img/v13.png">
-            <img class="imgh" src="img/v13.png">
+            <img src="img/Group.png" alt="" class="nxt">
+        </div>
+    </section>
+    <section class="work">
+        <div class="work-title1">
+            <h1 class="work-head">My Recent Works</h1>
```

Comment:
[Optional] Since you already have another `h1` tag above in the header, you may need to use the `h2`  in here for better accessibility practices.